src/core/public/core_app/errors/url_overflow_ui.tsx

```diff
@@ -0,0 +1,86 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+
+import { FormattedMessage } from '@kbn/i18n/react';
+import { EuiText, EuiHorizontalRule } from '@elastic/eui';
+
+import { IBasePath } from '../../http';
+import { IS_IE } from './url_overflow';
+
+export const UrlOverflowUi: React.FC<{ basePath: IBasePath }> = ({ basePath }) => {
+  return (
+    <React.Fragment>
+      <EuiText>
+        <p>
+          <FormattedMessage
+            id="core.ui.errorUrlOverflow.errorDescription"
+            defaultMessage="The URL for this dashboard is too long, and we can't display the dashboard."
```

Comment:
Nit: This error message can also occur in other cases (e.g. when editing a vega visualization). It's not dashboard specific .

Reply:
Fixed